src/common/transformations/src/transformations/common_optimizations/convert_pagedattn_inputs.cpp

```diff
@@ -109,7 +109,7 @@ ov::pass::ConvertPagedAttnInputs::ConvertPagedAttnInputs(const KVCacheConfig& co
         value_cache->set_element_type(value_cache_precision);
         bool status = false;
         if (pa_op->get_rt_info().count("num_k_heads") && pa_op->get_rt_info().count("k_head_size") &&
-            pa_op->get_rt_info().count("num_v_heads") && pa_op->get_rt_info().count("num_v_heads")) {
+            pa_op->get_rt_info().count("num_v_heads") && pa_op->get_rt_info().count("v_head_size")) {
```

Comment:
```inline constexpr const char* V_CHEAD_SIZE = "v_head_size```
Maybe we need such variable in header to avoid types or some helper function which get this RT info?

Reply:
@praasz Would you please use another PR to refactor the common optimization code? This PR is for new GPU feature of xattention.